overlord.pl

```diff
@@ -371,6 +376,13 @@ sub squidHasSwapouts
     return $mgrPage =~ /SWAPOUT_WRITING/;
 }
 
+# whether some of Squid jobs are of Rock::HeaderUpdater type
+sub squidHasRockHeaderUpdate
+{
+    my $mgrPage = &getCacheManagerResponse('jobs')->{content};
+    return $mgrPage =~ /Rock::HeaderUpdater/;
```

Comment:
Please tighten this match to look just at the reported AsyncJob::typeName field values, not the entire response. This may become important when we start looking for jobs with less distinct type names (or type name patterns).

Reply:
With 41212b2, we match the entire regex passed from Daft. Is this enough 'tightening'? Or we need a multiline match of two relevant of strings, i.e
```
job1089:
    type: 'Rock::HeaderUpdater'
```

Reply:
Moved to https://github.com/measurement-factory/squid-overlord/pull/4#discussion_r1365667059